Tests/Tactics/CSE.lean

```diff
@@ -57,17 +166,58 @@ info: [Tactic.cse.summary] CSE collected expressions: ⏎
+      [Tactic.cse.summary] (y + y):(Nat) [relevant? ✅] (unfold for subexpressions...)
+        [Tactic.cse.summary] (y):(Nat) [relevant? ✅] (unfold for subexpressions...)
+          [Tactic.cse.summary] Added new expr (...) with info ({ occs := 1, size := 1 }) (NOTE: can be large)
+            [Tactic.cse.summary] y
+        [Tactic.cse.summary] (y):(Nat) [relevant? ✅] (unfold for subexpressions...)
+          [Tactic.cse.summary] updated expr (...) with info ({ occs := 2, size := 1 }) (NOTE: can be large)
+            [Tactic.cse.summary] y
+        [Tactic.cse.summary] Added new expr (...) with info ({ occs := 1, size := 3 }) (NOTE: can be large)
+          [Tactic.cse.summary] y + y
+      [Tactic.cse.summary] (y + y):(Nat) [relevant? ✅] (unfold for subexpressions...)
+        [Tactic.cse.summary] (y):(Nat) [relevant? ✅] (unfold for subexpressions...)
+          [Tactic.cse.summary] updated expr (...) with info ({ occs := 3, size := 1 }) (NOTE: can be large)
+            [Tactic.cse.summary] y
+        [Tactic.cse.summary] (y):(Nat) [relevant? ✅] (unfold for subexpressions...)
+          [Tactic.cse.summary] updated expr (...) with info ({ occs := 4, size := 1 }) (NOTE: can be large)
+            [Tactic.cse.summary] y
+        [Tactic.cse.summary] updated expr (...) with info ({ occs := 2, size := 3 }) (NOTE: can be large)
+          [Tactic.cse.summary] y + y
+      [Tactic.cse.summary] Added new expr (...) with info ({ occs := 1, size := 7 }) (NOTE: can be large)
+        [Tactic.cse.summary] BitVec.ofNat (y + y) (y + y)
+    [Tactic.cse.summary] (x):(BitVec (y + (y + 0))) [relevant? ✅] (unfold for subexpressions...)
+      [Tactic.cse.summary] Added new expr (...) with info ({ occs := 1, size := 1 }) (NOTE: can be large)
+        [Tactic.cse.summary] x
+[Tactic.cse.summary] ⏭️ CSE eliminiating unprofitable expressions (#expressions:4):
+  [Tactic.cse.summary] ⏭️ Unprofitable { occs := 4, size := 1 } . (NOTE: can be large)
+    [Tactic.cse.summary] expr: y
+  [Tactic.cse.summary] ⏭️ Unprofitable { occs := 1, size := 1 } . (NOTE: can be large)
+    [Tactic.cse.summary] expr: x
+  [Tactic.cse.summary] ⏭️ Unprofitable { occs := 1, size := 7 } . (NOTE: can be large)
+    [Tactic.cse.summary] expr: BitVec.ofNat (y + y) (y + y)
+[Tactic.cse.summary] CSE summary of profitable expressions (#expressions:1):
+  [Tactic.cse.summary] 1) { occs := 2, size := 3 } (NOTE: can be large)
+    [Tactic.cse.summary] y + y
+[Tactic.cse.summary] CSE rewriting (#expressions:1):
+  [Tactic.cse.summary] ⌛ Generalizing hx1: x1 = ... (NOTE: can be large)
+    [Tactic.cse.summary] y + y
+  [Tactic.cse.summary] 💥 failed to generalize hx1 (NOTE: can be large)
+    [Tactic.cse.summary] tactic 'generalize' failed, result is not type correct
+          ∀ (x1 : Nat), BitVec.ofNat x1 x1 = x
+        y : Nat
+        x : BitVec (y + (y + 0))
+        ⊢ BitVec.ofNat (y + y) (y + y) = x
 -/
-#guard_msgs in example (x : BitVec 64) : (BitVec.ofNat 64 64) = x := by
```

Comment:
I had to make the example more complicated, because we now don't CSE literals like `64`.